### ISSUE TITLE
Blank tooltips for datetime64[ns]

### ISSUE THREAD

**Issue by ejain** [altair-viz/altair#1270]
This basic chart renders as expected, except that the "date" value in the tooltip is blank:

    import altair as alt
    from vega_datasets import data
    
    alt.Chart(data.stocks()).mark_point().encode(
        x='date',
        y='price',
        tooltip=['date', 'price', 'symbol']
    )

![chart](https://user-images.githubusercontent.com/1323903/49977272-83b8b700-fefa-11e8-9548-b00eda181fb6.png)

This can be fixed by specifying an explicit format with e.g. `alt.Tooltip('date', format='%d/%m/%Y')`, but it might make sense to have a default format?

**Comment by iaindillingham** [comment]
I've noticed that passing `timeUnit` when using a long-form `ToolTip` is a good workaround, albeit slightly more verbose.

```python
alt.Chart(data.stocks()).mark_point().encode(
    x='date',
    y='price',
    tooltip=[
        alt.Tooltip('date', timeUnit='yearmonthdate'),
        alt.Tooltip('price'),
        alt.Tooltip('symbol'),
    ],
)
```

**Comment by jakevdp** [comment]
A more concise way to do the same thing is
```python
tooltip=['yearmonthdate(date)', 'price', 'symbol']
```

**Comment by jakevdp** [comment]
Confirmed that this is fixed in Altair 3.0
